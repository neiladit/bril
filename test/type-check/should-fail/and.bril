main{

    v1: bool = const true;
    v2: bool = const 2;
    v3: bool = and v1 v2;
    print v3;

}

