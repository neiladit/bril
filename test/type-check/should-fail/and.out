line 3: assign non-bool value to bool var
const error!
